src/views/Receiver.vue

```diff
@@ -404,13 +404,12 @@ export default {
       this.NO_MORE = true;
     },
     bytesToSize(bytes) {
-      // TODO: Should this be 1024?
-      var k = 1000;
-      var sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+      let k = 1000;
+      let sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
       if (bytes === 0) {
         return '0 Bytes';
       }
-      var i = parseInt(Math.floor(Math.log(bytes) / Math.log(k)), 10);
+      let i = parseInt(Math.floor(Math.log(bytes) / Math.log(k)), 10);
       return (bytes / Math.pow(k, i)).toPrecision(3) + ' ' + sizes[i];
```

Comment:
Can you convert this string to ES6 templates?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals